modules/images-configuration-registry-mirror.adoc

```diff
@@ -0,0 +1,182 @@
+// Module included in the following assemblies:
+//
+// * openshift_images/image-configuration.adoc
+
+[id="images-configuration-registry-mirror_{context}"]
+= Configuring image registry repository mirroring
+Setting up container registry repository mirroring lets you:
+
+* Configure your {product-title} cluster to redirect requests
+to pull images from a repository on a source image registry and have
+it resolved by a repository on a mirrored image registry.
+* Identify multiple mirrored repositories for each target
+repository, to make sure that if one mirror is down, another
+can be used.
+
+Here are some of the attributes of repository mirroring in {product-title}:
+
+* Image pulls are resilient to registry downtimes
+* Disconnected clusters can request to pull
+images from critical locations (such as quay.io)
+and have registries behind a company firewall
+provide the requested images.
+* A particular order of registries is tried when
+an image pull request is made, with the permanent
+registry typically being the last one tried.
+
+Setting up repository mirroring can be done in the following ways:
+
+* At {product-title} installation time: By pulling container images needed
+by {product-title} and then bringing those images behind your company's
+firewall, you can install {product-title} into a datacenter that is
+disconnected from the Internet. See
+link:https://github.com/openshift/openshift-docs/blob/636ca5e931e8e3000349649d86f280a814de25c4/modules/installation-mirror-repository.adoc[Mirroring the {product-title} image repository]
```

Comment:
Remove this link

Reply:
I meant to replace this with the permanent location for setting up a mirrored repository at install time. I'll cut it for now, but leave this comment open if there is a permanent link I can use.